tests/integration/bond_test.py

```diff
@@ -173,11 +182,15 @@ def test_remove_bond_with_minimum_desired_state(eth1_up, eth2_up):
 def test_add_and_remove_bond_with_port_config(eth1_up, eth2_up):
     state = yaml.load(BOND99_PORT_YAML_BASE, Loader=yaml.SafeLoader)
     try:
-        libnmstate.apply(state)
+        apply_with_description(
+            "Add bond interface bond99 with bonding mode active-backup, "
+            "attach two ports eth1 and eth2 with ports config",
```

Comment:
What is "ports config" supposed to mean?

Reply:
It means the following ports config: 
```
interfaces:
- name: bond99
  type: bond
  state: up
  link-aggregation:
    mode: active-backup
    ports-config:
    - name: eth1
      priority: -1
      queue-id: 0
    - name: eth2
      priority: 2
      queue-id: 1
``` 

Maybe I should elaborate on the details of the `ports-config` in the description, but I also worried that this might not be natural if we include too many details in the description.

Reply:
How is an AI tool supposed to deduct the priority/queue-id values from just the statement "ports config"?

Reply:
Thanks for the catch, I will definitely make the description accurately detail the Nmstate state.